app/notifications/send_notifications.py

```diff
@@ -60,18 +122,6 @@ def send_notification_bypass_route(
     )
 
     if recipient_item is not None:
-        if not ('id_type' in recipient_item and 'id_value' in recipient_item):
-            current_app.logger.critical(
-                'Error in send_notification_bypass_route attempting to send notification id %s using recipient_item. '
-                'Must contain both "id_type" and "id_value" fields, but one or both are missing. recipient_item: %s',
-                notification.id,
-                recipient_item,
-            )
-            raise NotificationTechnicalFailureException(
-                'Error attempting to send notification using recipient_item. Must contain both "id_type" and "id_value"'
-                ' fields, but one or more are missing.'
-            )
-
         current_app.logger.info(
```

Comment:
Should this logger statement be moved under the rest of the `if recipient_item is not None` on line 95 of the update file?

Reply:
lines 95-105 is checking for the error state, this is referencing the call being made directly below it, so I think it makes sense to leave it here.